txma/template.yaml

```
@@ -223,4 +313,4 @@ Outputs:
     Description: The ARN of the audit event SQS queue DLQ
     Value: !GetAtt AuditEventDeadLetterQueue.Arn
     Export:
-      Name: !Sub ${AWS::StackName}-AuditEventDLQArn
+      Name: !Sub ${AWS::StackName}-AuditEventDLQArn
```

```suggestion
      Name: !Sub ${AWS::StackName}-AuditEventDLQArn
      
```
Restore the newline at the end of the file